includes/User.php

```diff
@@ -2648,22 +2648,7 @@ public function getGlobalAttribute( $attribute, $default = null ) {
 	 * @return bool
 	 */
 	private function shouldGetAttributeFromService( $attributeName ) {
```

Comment:
Yea, I was thinking about that too. Personally I'd like to leave it as is since I think it's more explicit about what's going on. If we just use `$this->isPublicAttribute( $attributeName );` the reader has to infer that pubic attributes mean that we use the attribute service. Keeping the 2 calls I think tells a better story with less guessing.